python/app/work_area_button.py

```diff
@@ -0,0 +1,303 @@
+# Copyright (c) 2015 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+
+from sgtk.platform.qt import QtCore, QtGui
+import sgtk
+
+
+
+class WorkAreaButtonDetailsArea(QtGui.QToolButton):
```

Comment:
The class name feels odd. Should this be called something like `SetWorkAreaButton`?